test/test_misb_0601.py

```diff
@@ -854,22 +855,22 @@ def test_PlatformAngleOfAttackFull(self):
         self.assertEqual(bytes(PlatformAngleOfAttackFull(example_value)), example_ls_packet)
         self.assertEqual(bytes(PlatformAngleOfAttackFull(example_ls_packet[2:])), example_ls_packet)
 
-    def test_PlatformSideslipAngleFull(self):
-        # Example value and packet per MISB ST 0601.11, Section 8 "Conversions and Mappings of Metadata Types".
-        example_value = 'x'
-        example_ls_packet = hexstr_to_bytes('5D 04 00 00 00 00')
-
-        from klvdata.misb0601 import PlatformSideslipAngleFull
-        self.assertEqual(bytes(PlatformSideslipAngleFull(example_value)), example_ls_packet)
-        self.assertEqual(bytes(PlatformSideslipAngleFull(example_ls_packet[2:])), example_ls_packet)
+#     def test_PlatformSideslipAngleFull(self):
+#         # Example value and packet per MISB ST 0601.11, Section 8 "Conversions and Mappings of Metadata Types".
+#         example_value = 'x'
+#         example_ls_packet = hexstr_to_bytes('5D 04 00 00 00 00')
+# 
+#         from klvdata.misb0601 import PlatformSideslipAngleFull
+#         self.assertEqual(bytes(PlatformSideslipAngleFull(example_value)), example_ls_packet)
+#         self.assertEqual(bytes(PlatformSideslipAngleFull(example_ls_packet[2:])), example_ls_packet)
 
     # Tag 94 (0x5E) MIIS Core Identifier (MISB ST 1204)
 
     # Tag 95 (0x5F) SAR Motion Imagery (MISB ST 1206) Local Set
 
     def test_TargetWidthExtended(self):
         # Example value and packet per MISB ST 0601.11, Section 8 "Conversions and Mappings of Metadata Types".
-        example_value = 13_898.5463
+        example_value = 13898.5463
```

Comment:
The '_' was added for readability, it does not impact the value when executed.

Reply:
I thought it was a mistake, because the original ( standar ) value is 13898.5463